### DISCUSSION TITLE
APISIX

### DISCUSSION THREAD

**Discussion by amyb12345** [docker/extension-ideas#26] [Ideas]
Note: This is a transfer from our initial method of collecting new extension requests. If you have additional information on how this will help you or other functionality you'd like to see, please add more details in the comments.

**Comment by tao12345666333** [comment]
Glad to see Apache APISIX listed here.

I think we can follow up on this thing.  The Docker Desktop extension has reached GA

**Comment by amyb12345** [comment]
@tao12345666333 Great to hear that you can follow up.  Don't hesitate to reach out to the Docker team if you have any questions or need any support. We can be reached on our [community Slack](http://dockr.ly/comm-slack) in #extensions or on [our SDK repo](https://github.com/docker/extensions-sdk).

Check out our [getting started page](https://www.docker.com/developers/sdk/) for more resources.
